templates/cloudflare-workers/app/root.tsx

```diff
@@ -1,10 +1,16 @@
+import type { LinksFunction } from "@remix-run/cloudflare";
 import {
   Links,
   Meta,
   Outlet,
   Scripts,
   ScrollRestoration,
 } from "@remix-run/react";
+import stylesheet from "~/tailwind.css?url";
+
+export const links: LinksFunction = () => [
+  { rel: "stylesheet", href: stylesheet },
+];
```

Comment:
Is there any reason to not just use a side-effect css import with tailwind?

Reply:
Since Vite is now the default, I prefer the side-effect import for tailwind. It's more idiomatic.

Reply:
Also makes it easier to remove if you don't want to use tailwind.

Updated!